packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2611,6 +2614,7 @@ class InputDecoration {
     this.alignLabelWithHint,
     this.constraints,
   }) : assert(!(label != null && labelText != null), 'Declaring both label and labelText is not supported.'),
+       assert(!(error != null && errorText != null), 'Declaring both error and errorText is not supported.'),
```

Comment:
If we add `validatorBuilder`, we should have an assertion similar to this in FormField for it and `validator`.